cmd/spire-server/cli/entry/show_test.go

```diff
@@ -5,9 +5,9 @@ import (
 
 	"github.com/golang/mock/gomock"
 	"github.com/spiffe/spire/pkg/common/util"
-	"github.com/spiffe/spire/proto/spire/api/registration"
-	"github.com/spiffe/spire/proto/spire/common"
-	mock_registration "github.com/spiffe/spire/test/mock/proto/api/registration"
+	"github.com/spiffe/spire/proto/spire/api/server/entry/v1"
+	"github.com/spiffe/spire/proto/spire/types"
+	mock_entry "github.com/spiffe/spire/test/mock/proto/api/entry"
```

Comment:
I think that there is a general consensus of trying to avoid this kind of mocks in the new tests that we write.
I would suggest to have tests using fake service implementations. In this case, we can have a fake entry service. Examples of how tests have been written this way are the tests for the `spire-server agent` and `spire-server bundle` commands. It would be great if we can have that kind of testing here also.
What do you think?

Reply:
sounds good!